working/0546-patterns/patterns-feature-specification.md

```diff
@@ -1738,6 +1799,32 @@ It is a compile-time error if:
     destructuring and variable binding won't fail to match (though it might
     throw a runtime exception from implicit downcasts from `dynamic`).*
 
+If `p` has required type `T` and `M` is not a subtype of `T` but is assignable,
+then an implicit coercion or cast is inserted before the pattern accesses the
```

Comment:
It's not clear to me what it means for a pattern to access the value.  Maybe it might be clearer to say "is inserted at the point that the pattern destructures, matches, or binds the matched value"?